pkg/cmd/roachtest/drop.go

```diff
@@ -74,7 +108,43 @@ gc:
 '`
 			run(stmtZone)
 
-			// TODO(tschottdorf): assert that the disk usage drops to "near nothing".
+			// We're waiting a maximum of 5 minutes to makes sure that the drop operations clear the disk.
+			for i := 0; i < 5; i++ {
+				allNodesSpaceCleared := true
+				for j := 1; j <= nodes; j++ {
+					out, err := c.RunWithBuffer(ctx, c.l, j, fmt.Sprintf("du -sc {store-dir:%d}", j))
```

Comment:
Why `-c`? I thought `du -s` was the right one.

And I think you could cheat a little and just run

`du -s {store-dir:%d} | grep -oE '^[0-9]+`

and not bother with all the byte extraction below.